lib/go/edgecontext/edgecontext.go

```diff
@@ -43,13 +43,20 @@ const LoIDPrefix = "t2_"
 // e.g. en, en_US
 var LocaleRegex = regexp.MustCompile(`^[a-z]{2}(_[A-Z]{2})?$`)
 
+// UnifiedLocaleRegex validates locale codes format.
+// The same as LocaleRegex, but language and region specifier separated by a hyphen.
+var UnifiedLocaleRegex = regexp.MustCompile(`^[a-z]{2}(\-[A-Z]{2})?$`)
```

Comment:
I do see some of the languages have 3 instead of 2 characters: https://en.wikipedia.org/wiki/IETF_language_tag#List_of_major_primary_language_subtags

we probably want to fix `LocaleRegex` as well.

Reply:
Good catch, thank you! Will update.
cc @ZackMcBride